src/flupy/cli/cli.py

```diff
@@ -66,8 +65,6 @@ def main(argv: Optional[List[str]] = None) -> None:
     if _file:
         _ = flu(read_file(_file)).map(str.rstrip)
     else:
-        # Do not raise exception for Broken Pipe
-        signal(SIGPIPE, SIG_DFL)
```

Comment:
this is necessary to resolve broken pipe exceptions on unix systems

how about
```py
import platform
...

if platform.system() != 'Windows':
  from signal import SIG_DFL, SIGPIPE, signal
  signal(SIGPIPE, SIG_DFL)
...
```

Reply:
Sure, at that point probably best to wrap with `try/catch ImportError` instead of explicitly checking platform I think... that way it'll continue to work on any other platforms that don't have `SIGPIPE`.

Do you have a way to reproduce the broken pipe exception on Unix? I tried a couple different ways but never got it to happen.

Reply:
try:
```py
flu 'flu(count())' -i 'itertools:count' | head -n 1
```
<img width="312" alt="Screenshot 2024-04-30 at 9 44 36 AM" src="https://github.com/olirice/flupy/assets/12958657/0da090c6-ed22-4f9d-939b-3a12a12c3b77">